src/CfgEditor/Sections.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export class Sections {
+  static onecc: string[] = [
+    "onecc",
+    "one-import-tf",
+    "one-import-tflite",
+    "one-import-bcq",
+    "one-import-onnx",
+    "one-optimize",
+    "one-quantize",
+    "one-codegen",
+    "one-profile",
+    "one-plus",
+    "one-minus",
+  ];
+  static edgetpu: string[] = [
```

Comment:
(Suggestions) It would be better to separate each backend section string into each file later.